src/renderer/App.css

```
@@ -163,6 +163,12 @@
   touch-action: pan-x;
 }
 
+/* Chromium may claim the first finger as a pan before the second finger lands,
+   cancelling the pointer stream before the player can recognize a pinch. */
+.app.capacitorTabs :deep(.ftVideoPlayer.videoZoomTouchEnabled:not(.scrollMiniPlayer)) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)